docs/installation/dns_setup.md

```diff
@@ -1,157 +1,201 @@
+:::
 
-1. Find the IP address of the ingress controller
-2. Setup your DNS
+For simplicity finish your DNS setup before installing Epinio.
+You need to point your desired domain to the IP address of your Ingress controller.
+The two steps in the process are described in the following sections:
 
-> **NOTE:** For development or demo environments, [an easy wildcard DNS setup can be used](wildcardDNS_setup.md) instead.
+1. [Find](#find-ip-for-ic) the IP address of the ingress controller
+1. [Configure](#config-dns) your DNS
 
-## Ingress controller IP address
+:::note
 
-Most Kubernetes clusters are running a "load balancer" service which is responsible of assigning
-IP addresses to load balanced services created on the cluster. Ingress controllers
-are such services (e.g. Traefik) and they should have an externally accessible IP address.
-This is needed for any Ingress resource to work. You can find the load balancer IP
-address of any service using `kubectl`. E.g.
+For development or demo environments, [an easy wildcard DNS setup can be used](wildcardDNS_setup.md).
+
+:::
+
+## Find the IP address of the ingress controller {#find-ip-for-ic}
+
+Most Kubernetes clusters run a "load balancer" service.
+It is responsible for assigning IP addresses to services, that are load balanced, created on the cluster.
+Ingress controllers are such services (for example, Traefik) so should have an externally accessible IP address.
+This is needed for any Ingress resource to work.
+
+You can find the load balancer IP address of any service using `kubectl`. For example:
 
 ```
 kubectl get svc -n kube-system traefik -o jsonpath={@.status.loadBalancer.ingress}
 ```
+<!-- TODO: I would like a sample output from the previous command showing the IP address field.-->
 
-Depending on the load balancer, the result of this command may have different fields populated, most usually `ip`.
-Keep this information available for the next step where you will configure your DNS.
+Depending on the load balancer, the result of this command may have different fields populated.
+We are interested in the `ip` field.
+You use the IP field in the next step to configure your DNS.
 
-## Configure your DNS
+## Configure your DNS {#config-dns}
 
-Given you own the domain **"example.com"**, you would configure a subdomain e.g. **"test.example.com"** for Epinio.
-Now you will have to configure your DNS in a way, that any request towards "test.example.com" will resolve to the address found in the previous step.
+Assuming you own the domain `example.com`, you would configure a subdomain, for example, `test.example.com` for Epinio.
+You can now configure your DNS so that any request for `test.example.com`
+resolves to the address you obtained in the previous step.
 
-- test.example.com => "INGRESS-IP"
-- \*.test.example.com => "INGRESS-IP"
+<!--TODO: We have described what needs to be configured in DNS, but not actually shown how to do it.
+Do we need to? Or am I missing the point.-->
```

Comment:
The commands/actions strongly depend on the exact DNS vendor used. I.e. `AWS Route 53` is different from whatever `Google Cloud` offers, is different from `MicroSoft Azur`, is different from ...

Reply:
And we do seem to show it for Route 53, Azure, and Bind, in the examples coming after.